packages/shared/src/components/modals/common/Modal.tsx

```diff
@@ -122,9 +127,25 @@ export function Modal({
         onTrackNext,
         onTrackPrev,
         isDrawer: isDrawerOpen,
+        isForm,
       }}
     >
-      {children}
+      <ConditionalWrapper
+        condition={!!formProps && isMobile}
```

Comment:
can't we reuse `isForm` from above here?

Reply:
Ahh, good catch. Definitely must reuse that here.